### PR TITLE
Clustername

### PR DESCRIPTION
**Pull request by reneluria**
Hello,
This allows to specify the cluster domain name in the deployment of the operator instead of the default cluster.local
This parameter is already available in the helm chart but not in the operator

It is meant to replace PR #81 in the way that as the cluster domain is the same for the whole cluster, it does not make sense to specify it for each minio instance created by the operator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ This will create all relevant resources required for the Operator to work. Here
...
 
+To specify the cluster domain instead of the default `cluster.local`, add an environment variable to the pod spec on the deployment.
+
+```
+env:
+    - name: CLUSTER_DOMAIN
+      value: mycluster.mydomain
+```
+
+Or modify the file `minio-operator-with-cluster-name.yaml`
```
Comment (nitisht) [inline README.md:43]: Can we add this under the `Environment variables` section

File: minio-operator-with-cluster-name.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        name: minio-operator
+    spec:
+      serviceAccountName: minio-operator-sa
+      containers:
+        - name: minio-operator
+          image: minio/k8s-operator:1.0.8
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: CLUSTER_DOMAIN
+              value: mycluster.mydomain
```
Comment (nitisht) [inline minio-operator-with-cluster-name.yaml:139]: We can simply add 

```yaml
          env:
            - name: CLUSTER_DOMAIN
              value: mycluster.mydomain
```

In the `minio-operator.yaml` file and comment it out. Too many examples sometime get confusing

**Comment by reneluria** [comment]
> Thanks for the PR @reneluria looks good. Couple of doc comments, we can then take it in

Nice, how does it work ? you make the changes you mentioned here or do you need me to apply them first ?

**Comment by nitisht** [comment]
> Nice, how does it work ? you make the changes you mentioned here or do you need me to apply them first ?

It'd be great if you can apply the changes

File: README.md (hunk 1 above)
Comment (reneluria) [reply README.md]: Ok, good idea

File: minio-operator-with-cluster-name.yaml (hunk 2 above)
Comment (reneluria) [reply minio-operator-with-cluster-name.yaml]: Great, done.

**Pull request merged**